WalletWasabi.Fluent/Views/Wallets/History/HistoryTable.axaml

```diff
@@ -0,0 +1,120 @@
+             x:CompileBindings="True"
+             x:Class="WalletWasabi.Fluent.Views.Wallets.History.HistoryTable">
+  <DockPanel LastChildFill="True">
+
+    <DockPanel DockPanel.Dock="Top" HorizontalAlignment="Stretch">
+
+      <!-- Hide CJ switch -->
+      <StackPanel Orientation="Horizontal" Spacing="10" DockPanel.Dock="Right">
+        <TextBlock Text="Hide CoinJoin transactions" VerticalAlignment="Center" Opacity="0.6"/>
+        <ToggleSwitch IsChecked="{Binding HideCoinJoin}" OnContent="{x:Null}" OffContent="{x:Null}" VerticalAlignment="Center"/>
+      </StackPanel>
+
+      <!-- Title -->
+      <Label Content="Transaction History" DockPanel.Dock="Left" FontWeight="Bold" FontSize="16"/>
+    </DockPanel>
+
+    <DataGrid Items="{Binding Histories}" DockPanel.Dock="Bottom">
+      <DataGrid.Styles>
+        <Style Selector="PathIcon">
+          <Setter Property="Width" Value="20"/>
+          <Setter Property="Height" Value="20"/>
+          <Setter Property="VerticalAlignment" Value="Center"/>
+        </Style>
+        <Style Selector="TextBlock">
+          <Setter Property="VerticalAlignment" Value="Center"/>
+        </Style>
+        <Style Selector="StackPanel">
+          <Setter Property="Orientation" Value="Horizontal" />
+          <Setter Property="Spacing" Value="10" />
+          <Setter Property="Margin" Value="10 0" />
+        </Style>
+      </DataGrid.Styles>
+
+      <DataGrid.Columns>
+
+        <!-- Date -->
+        <DataGridTextColumn Header="Date"
+                            x:DataType="history:HistoryItemViewModel"
+                            Binding="{Binding Date}"
+                            Width="*" />
+
+        <!-- Labels -->
+        <DataGridTemplateColumn Header="Labels"
+                                Width="2*"
+                                SortMemberPath="Labels">
+          <DataGridTemplateColumn.CellTemplate>
+            <DataTemplate>
+              <c:TagsBox x:DataType="history:HistoryItemViewModel" Items="{Binding Labels}" IsReadOnly="True" Margin="10 0" />
+            </DataTemplate>
+          </DataGridTemplateColumn.CellTemplate>
+        </DataGridTemplateColumn>
+
+        <!-- Outgoing -->
+        <DataGridTemplateColumn Header="Outgoing"
+                                Width="*"
+                                SortMemberPath="OutgoingAmount">
+          <DataGridTemplateColumn.CellTemplate>
+            <DataTemplate>
+              <StackPanel x:DataType="history:HistoryItemViewModel"
+                          IsVisible="{Binding !!OutgoingAmount}">
+                <LayoutTransformControl VerticalAlignment="Center">
+                  <LayoutTransformControl.LayoutTransform>
+                    <RotateTransform Angle="180" />
+                  </LayoutTransformControl.LayoutTransform>
+                  <PathIcon Data="{StaticResource chevron_down_circle_regular}" Opacity="0.6" />
+                </LayoutTransformControl>
```

Comment:
I haven't found the `up` version of this icon? Any simpler solution? @jmacato

Reply:
M10 20C4.47715 20 0 15.5228 0 10C0 4.47715 4.47715 0 10 0C15.5228 0 20 4.47715 20 10C20 15.5228 15.5228 20 10 20ZM10 18.5C14.6944 18.5 18.5 14.6944 18.5 10C18.5 5.30558 14.6944 1.5 10 1.5C5.30558 1.5 1.5 5.30558 1.5 10C1.5 14.6944 5.30558 18.5 10 18.5ZM14.5303 12.0303C14.2374 12.3232 13.7626 12.3232 13.4697 12.0303L10 8.56066L6.53033 12.0303C6.23744 12.3232 5.76256 12.3232 5.46967 12.0303C5.17678 11.7374 5.17678 11.2626 5.46967 10.9697L9.46967 6.96967C9.76256 6.67678 10.2374 6.67678 10.5303 6.96967L14.5303 10.9697C14.8232 11.2626 14.8232 11.7374 14.5303 12.0303Z

Reply:
Awesome! thanks